form.js

```diff
@@ -0,0 +1,21 @@
+const containerform = document.querySelector('.input');
+const error = document.querySelector('#error');
+containerform.addEventListener('submit', (e) => {
+  let right = true;
+  const Demail = document.querySelector('#Email').value;
+  const emailArray = Demail.split('');
+  for (let i = 0; i < emailArray.length; i += 1) {
+    if (
+      Demail !== Demail.toLowerCase()
+        && emailArray[i] !== '@' && emailArray[i] !== '.'
+    ) {
+      right = false;
+    }
+  }
+  if (right) {
+    error.innerHTML = '';
+  } else {
+    e.preventDefault();
+    error.innerHTML = 'Please enter your email in lower case only';
```

Comment:
- [ ] This error message should show up close to the "submit" button, as for now it does not, please fix this so the error message displayed on the form when there is capital letters on the email